virttest/utils_misc.py

```diff
@@ -3987,3 +3989,30 @@ def setup_remote(self):
         logging.debug("To check remote boolean value: %s", boolean_curr)
         if boolean_curr != self.remote_bool_value:
             raise exceptions.TestFail(result.stderr.strip())
+
+
+def get_model_features(modelname):
+    """
+    /usr/share/libvirt/cpu_map.xml defines all CPU models.
+    One CPU model is a set of features.
+    This funtion is to get features of one specific model.
+    """
+    features = []
+    conf = "/usr/share/libvirt/cpu_map.xml"
+    if not os.path.exists(conf) or not os.path.isfile(conf):
```

Comment:
why need 2 check? isfile is not enough?

Reply:
After saw your following codes, I think you can add try in front of the func and then for the situation that file do not exist, you can catch FileNotFoundError like your following catch

Reply:
isfile is enough.